src/main/java/dansplugins/factionsystem/services/PlayerService.java

```diff
@@ -0,0 +1,47 @@
+package dansplugins.factionsystem.services;
+
+import dansplugins.factionsystem.MedievalFactions;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.ConsoleCommandSender;
+
+import java.util.List;
+
+public class PlayerService {
+
+    public String getMessageType(String OldType, String NewType) {
```

Comment:
Variables should start with lowercase (i.e. `oldType` and `newType`)

Reply:
Oki, i'll change it in next push